example/examples/CachedMap.js

```diff
@@ -18,7 +18,9 @@ class CachedMap extends React.Component {
   constructor(props) {
     super(props);
 
-    const ds = new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 !== r2 });
+    const ds = new ListView.DataSource({
```

Comment:
not in the scope of this PR, but maybe worth migrating away from all ListViews

Reply:
(in a separate PR)

Reply:
I'll try to make a PR for that, thnx